user/user.go

```diff
@@ -84,6 +85,35 @@ type Capabilities struct {
 	ChatReferenceID        bool `ocscapability:"chat-reference-id"`
 }
 
+// RoomInfo contains information about a room
+type RoomInfo struct {
+	Token                 string                  `json:"token"`
+	Name                  string                  `json:"name"`
+	DisplayName           string                  `json:"displayName"`
+	SessionID             string                  `json:"sessionId"`
+	ObjectType            string                  `json:"objectType"`
+	ObjectID              string                  `json:"objectId"`
+	Type                  int                     `json:"type"`
```

Comment:
Could be nice to make a type for this to make it easier for developers but that can be done at another time.